# Legenda

## Preto = Títulos

## <font color='blue'>Azul = Comentários</font>

## <font color='red'>Vermelho = Respostas</font>

# Importação das bibliotecas

In [1]:
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
import seaborn as sns

%matplotlib inline

# Leitura dos Data Frames

In [2]:
faturamento = pd.read_csv('resumo-faturamento-price-info.csv',delimiter=';')
faturamento.head()

IDIMOVEL  01/08/2019  01/09/2019  01/10/2019  01/11/2019  01/12/2019  \
0     20927        4974        9723        8373        5184        7644   
1     26364          84           0          84           0        1010   
2     37265         900         297           0        1386        2692   
3     82094           0           0           0           0        1125   
4     83547       85000       55000      150000      145000      155000   

   01/01/2020  
0        3861  
1           0  
2        3720  
3        1935  
4      155000

In [3]:
ocupacao = pd.read_csv('resumo-ocupacao-price-info.csv',delimiter=';')
ocupacao.head()

IDIMOVEL  01/08/2019  01/09/2019  01/10/2019  01/11/2019  01/12/2019  \
0     20927          13          26          24          16          25   
1     26364           1           0           1           0          10   
2     37265          10           3           0          14          19   
3     82094           0           0           0           0          17   
4     83547          17          11          30          29          31   

   01/01/2020  
0          14  
1           0  
2          20  
3          31  
4          31

# O ID que teve o maior faturamento

## <font color='blue'>Calculando o total de cada ID</font>

In [29]:
meses = ['01/08/2019','01/09/2019','01/10/2019','01/11/2019','01/12/2019','01/01/2020']
soma = 0

for i in meses:
    soma = soma + faturamento[i]

faturamento['TOTAL'] = soma  
faturamento.head()

IDIMOVEL  01/08/2019  01/09/2019  01/10/2019  01/11/2019  01/12/2019  \
0     20927        4974        9723        8373        5184        7644   
1     26364          84           0          84           0        1010   
2     37265         900         297           0        1386        2692   
3     82094           0           0           0           0        1125   
4     83547       85000       55000      150000      145000      155000   

   01/01/2020   TOTAL  
0        3861   39759  
1           0    1178  
2        3720    8995  
3        1935    3060  
4      155000  745000

## <font color='blue'>Localizando o máximo</font>

In [30]:
faturamento[faturamento['TOTAL'] == faturamento['TOTAL'].max()]

IDIMOVEL  01/08/2019  01/09/2019  01/10/2019  01/11/2019  01/12/2019  \
4     83547       85000       55000      150000      145000      155000   

   01/01/2020   TOTAL  
4      155000  745000

## <font color='red'>O imóvel que mais faturou foi o 83547 com um total de R$745000</font>

# O ID que teve o menor faturamento

In [31]:
faturamento[faturamento['TOTAL'] == faturamento['TOTAL'].min()]

IDIMOVEL  01/08/2019  01/09/2019  01/10/2019  01/11/2019  01/12/2019  \
5       96939           0           0           0           0           0   
11     122504           0           0           0           0           0   
12     125064           0           0           0           0           0   
13     130736           0           0           0           0           0   
34     291437           0           0           0           0           0   
68     693457           0           0           0           0           0   
72     711032           0           0           0           0           0   
75     748321           0           0           0           0           0   
84     780520           0           0           0           0           0   
85     780550           0           0           0           0           0   
104    942411           0           0           0           0           0   
113    982402           0           0           0           0           0   
134   1564315           0           0           0           0           0   
183   2083416           0           0           0           0           0   
186   2103758           0           0           0           0           0   
198   2137779           0           0           0           0           0   
218   2223263           0           0           0           0           0   
240   4108729           0           0           0           0           0   
255   4293449           0           0           0           0           0   
262   4330674           0           0           0           0           0   
286   4511690           0           0           0           0           0   
296   4587329           0           0           0           0           0   
347  10363087           0           0           0           0           0   
348  10366668           0           0           0           0           0   
349  10367722           0           0           0           0           0   
352  16251833           0           0           0           0           0   
365  16294535           0           0           0           0           0   
369  16298244           0           0           0           0           0   
376  16323237           0           0           0           0           0   
387  16331050           0           0           0           0           0   
391  16333503           0           0           0           0           0   
392  16333649           0           0           0           0           0   
395  16849257           0           0           0           0           0   
396  16851036           0           0           0           0           0   
417  16897092           0           0           0           0           0   
457  31242632           0           0           0           0           0   
469  32982749           0           0           0           0           0   
472  32991670           0           0           0           0           0   
476  32994399           0           0           0           0           0   
479  33010577           0           0           0           0           0   

     01/01/2020  TOTAL  
5             0      0  
11            0      0  
12            0      0  
13            0      0  
34            0      0  
68            0      0  
72            0      0  
75            0      0  
84            0      0  
85            0      0  
104           0      0  
113           0      0  
134           0      0  
183           0      0  
186           0      0  
198           0      0  
218           0      0  
240           0      0  
255           0      0  
262           0      0  
286           0      0  
296           0      0  
347           0      0  
348           0      0  
349           0      0  
352           0      0  
365           0      0  
369           0      0  
376           0      0  
387           0      0  
391           0      0  
392           0      0  
395           0      0

## <font color='red'>Diversos imóveis não foram alugados no período analisado</font>

# A média de faturamento dos IDs

In [32]:
faturamento['TOTAL'].mean()

23177.258517034068

## <font color='red'>A média dos faturamentos foi de R$23177.26 por ID no período de 6 meses</font>

# Qual mês apresentou o maior (e o menor) faturamento médio

In [40]:
faturamento_medio = {}

for i in meses:
    faturamento_medio[i] =faturamento[i].mean()
    print(i,faturamento_medio[i])

01/08/2019 1612.0240480961925
01/09/2019 1349.5250501002004
01/10/2019 2438.068136272545
01/11/2019 3117.184368737475
01/12/2019 6282.701402805611
01/01/2020 8377.755511022044


## <font color='red'>O mês de janeiro apresentou o maior faturamento e setembro o menor faturamento médio no mês</font>

# Qual a média de taxa de ocupação para Janeiro? e para Outubro?

In [44]:
ocupacao['01/01/2020'].mean()/31

0.5362984032581292

In [45]:
ocupacao['01/10/2019'].mean()/31

0.22069946344301508

## <font color='red'>O mês de Janeiro teve 54% de média de ocupação enquanto o mês de outubro somente 22%</font>